src/components/SectionTitle/SectionTitle.tsx

```diff
@@ -69,14 +70,13 @@ export default class SectionTitle extends React.Component<
 
         let headingModifiers = linkUrl ? ["has-link"] : [];
         return (
-            <div className={bem(baseClass, modifiers, blockName)}>
+            <div className={bem(baseClass, modifiers, blockName, [className])}>
```

Comment:
this is eventually being deprecated right

Reply:
This component? Yeah, I think we can safely bundle it as a `Heading` story.